Neos.Flow/Classes/Security/AccountIdentifier.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Flow\Security;
```

Comment:
Maybe a sub namespace (e.g. "Account") makes sense for the new classes? (not sure about it)